src/main/java/org/apache/commons/beanutils2/sql/converters/SqlTimeConverter.java

```diff
@@ -62,12 +62,6 @@ protected Class<Time> getDefaultType() {
     }
 
     /**
-     * Gets a {@code DateFormat} for the Locale.
```

Comment:
This seems to be going in the wrong direction. You want to fill in the TODO, not blow away the whole Javadoc comment.